package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "snyk-php-plugin",
   "description": "Snyk CLI PHP plugin",
-  "main": "lib/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "lint": "eslint -c .eslintrc lib test",
-    "test": "tap test/*.test.js -R spec",
-    "semantic-release": "semantic-release"
+    "build": "tsc",
+    "build-watch": "tsc -w",
+    "lint": "tslint -p tsconfig.json",
+    "prepare": "npm run build",
+    "semantic-release": "semantic-release",
+    "test": "npm run lint && npm run unit-test",
+    "unit-test": "tap -Rspec test/*.test.ts --timeout=300"
```

Comment:
For testing are we not moving away from tap and using jest instead?

Reply:
@lwywoo I think we are. But this will take some time and we want to release feature this sprint